src/lib/FlightTasks/tasks/FlightTaskAutoLine.cpp

```diff
@@ -45,110 +45,13 @@ using namespace matrix;
 
 bool FlightTaskAutoLine::activate()
 {
-	bool ret = FlightTaskAuto::activate();
-	_reset();
+	bool ret = FlightTaskAutoMapper::activate();
+	//_reset();
 	return ret;
```

Comment:
I guess we can remove the comment and just put `return FlightTaskAuto::activate();`